### PR TITLE
AC Automated Test - LRAC-9062, LRAC-9085, LRAC-9086, LRAC-9087, LRAC-9088, LRAC-9089 | master

### PR DESCRIPTION
**Pull request by mateusralv**
https://issues.liferay.com/browse/LRAC-9062
https://issues.liferay.com/browse/LRAC-9085
https://issues.liferay.com/browse/LRAC-9086
https://issues.liferay.com/browse/LRAC-9087
https://issues.liferay.com/browse/LRAC-9088
https://issues.liferay.com/browse/LRAC-9089

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro [hunk 1]
```diff
@@ -615,6 +615,19 @@ definition {
 		}
 	}
 
+	@summary = "The macro deletes all users that have pending status."
+	macro deletePendingAllUsers {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro:619]: hi @mateusralv , could you please rename this macro to: `deleteAllPendingUsers`?

**Comment by alexcandidos** [comment]
hi @ViniLopes20  , thanks for helping to review it.

Hi @mateusralv , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**